source/linux/Overview/Processor_SDK_Linux_create_SD_card.rst

```diff
@@ -54,7 +54,10 @@ Create SD Card using balenaEtcher
 ----------------------------------
 
 1.  Download the default bootable SD card image (WIC file) available on the release page as
-    :file:`tisdk-default-image-<machine>.wic.xz`
+
+.. parsed-literal::
```

Comment:
@sinha-shreyash I think you need to indent this as well.

For the numbered list to continue below with (2.) everything under (1.)  has to be indented for it to be rendered under (1.)

Same comment for other places in the file.